frontend/src/app/page.tsx

```diff
@@ -1,103 +1,166 @@
+              <div className={"flex items-center justify gap-3 mb-4"}>
+                  <span className={"text-white font-bold text-lg"}>
+                      AI
+                  </span>
+              </div>
+              <h1 className={"text-3xl font-bold text-gray-800 dark:text-white"}>
+                  Learning Companion
+              </h1>
+          </div>
+      </div>
+        <div className={"flex justify-center mb-8"}>
+            <div className={"bg-white dark:bg-gray-800 rounded-lg p-1 shadow-lg"}>
+                <button
+                    onClick={() => setActiveTab('upload')}
+                    className={`px-6 py-2 rounded-md font-medium transition-all ${
+                        activeTab === 'upload'
+                            ? 'bg-blue-500 text-white shadow-md'
+                            : 'text-gray-600 dark:text-gray-300 hover:text-blue-500'
+                    }`}
+                >
+                    Upload Notes
+                </button>
+                <button
+                    onClick={() => setActiveTab('summary')}
+                    className={`px-6 py-2 rounded-md font-medium transition-all ${
+                        activeTab === 'summary'
+                            ? 'bg-blue-500 text-white shadow-md'
+                            : 'text-gray-600 dark:text-gray-300 hover:text-blue-500'
+                    }`}
+                    disabled={!summary && !isLoading}
+                >
+                    Summary
+                </button>
+            </div>
+        </div>
+        <div className="max-w-4xl mx-auto">
+            {activeTab === 'upload' && (
+                <div className="bg-white dark:bg-gray-800 rounded-xl shadow-xl p-8">
+                    <h2 className="text-2xl font-semibold text-gray-800 dark:text-white mb-6">
+                        Upload Your Notes
+                    </h2>
+
+                    <div className="space-y-6">
+                        <div>
+                            <label className="block text-sm font-medium text-gray-700 dark:text-gray-300 mb-2">
+                                Paste your lecture notes, slides, or study material below:
+                            </label>
+                            <textarea
+                                value={notes}
+                                onChange={(e) => setNotes(e.target.value)}
+                                placeholder="Paste your notes here... You can include lecture transcripts, slide content, textbook excerpts, or any study material you'd like to transform into learning resources."
+                                className="w-full h-64 p-4 border border-gray-300 dark:border-gray-600 rounded-lg resize-none focus:ring-2 focus:ring-blue-500 focus:border-transparent dark:bg-gray-700 dark:text-white"
+                            />
+                        </div>
+
+                        <div className="flex items-center justify-between">
```

Comment:
```suggestion
                        <div className="flex items-center justify-between gap-3">
```

Very minor change that gives the same spacing between the buttons and char count.